src/Illuminate/Auth/MustVerifyEmail.php

```diff
@@ -47,4 +47,14 @@ public function getEmailForVerification()
     {
         return $this->email;
     }
+
+    /**
+     * Get the name of the "email verified at" column.
+     *
+     * @return string
+     */
+    public function getEmailVerifiedAtColumn(): string
+    {
+        return defined(static::class.'::EMAIL_VERIFIED_AT') ? static::EMAIL_VERIFIED_AT : 'email_verified_at';
```

Comment:
Just a method to overwrite would be enough here I think.

```suggestion
        return 'email_verified_at';
```